pdf/sailfishofficepdfplugin.cpp

```diff
@@ -35,4 +36,8 @@ void SailfishOfficePDFPlugin::registerTypes(const char *uri)
     qmlRegisterType<PDFCanvas>(uri, 1, 0, "Canvas");
     qmlRegisterType<PDFLinkArea>(uri, 1, 0, "LinkArea");
     qmlRegisterType<PDFSelection>(uri, 1, 0, "Selection");
+    qmlRegisterType<PDFAnnotation>(uri, 1, 1, "Annotation");
+    qmlRegisterType<PDFTextAnnotation>(uri, 1, 1, "TextAnnotation");
+    qmlRegisterType<PDFCaretAnnotation>(uri, 1, 1, "CaretAnnotation");
+    qmlRegisterType<PDFHighlightAnnotation>(uri, 1, 1, "HighlightAnnotation");
```

Comment:
Ok, I think you're right. Since I was adding new QML object in the library, I increased the version number, but SailfishOffice being the only user of this library, it makes no sense to bother with the versioning. Switched back to 1.0.